examples/shared_vpc/main.tf

```diff
@@ -41,15 +41,18 @@ provider "random" {
 /******************************************
   Host Project Creation
  *****************************************/
+# [START vpc_shared_vpc_host_project_create]
 module "host-project" {
-  source                         = "../../"
+  source                         = "terraform-google-modules/terraform-google-project-factory/google//modules/shared_vpc_access"
```

Comment:
This isn't the right source. What are you trying to show with this example?

Reply:
I'm trying to add some Terraform tabs to this page: https://cloud.google.com/vpc/docs/provisioning-shared-vpc#setting_up

Reply:
Got it. This isn't the right example for that, as this actually *creates* the host project. Is that what you want to do?

If you just want to activate Shared VPC on an existing project, a new example would be needed.

Reply:
Got it. Thanks for the guidance. I'm closing this PR and starting over with a new example.